cli/internal/cmd/upgradeapply_test.go

```diff
@@ -30,6 +30,8 @@ import (
 
 func TestUpgradeApply(t *testing.T) {
 	someErr := errors.New("some error")
+	azureSEVSNP, err := config.DefaultForAzureSEVSNP()
+	require.NoError(t, err)
```

Comment:
Since this function makes real API calls now, we should not use it in unit tests, but rather have a function that returns placeholder values or use an empty struct